test/README.md

```diff
@@ -47,7 +47,7 @@ These environments execute all tests using [pytest], which supports its own simp
 
 ## Notebook environments
 
-The `notebook` and `py##-notebook` environments invoke [nbmake] to ensure that all the Jupyter notebooks in the [`docs/`](/docs/tutorials) directory execute successfully.
+The `notebook` and `py##-notebook` environments invoke [nbmake] to ensure that all Jupyter notebooks in the [`docs/`](/docs/) directory execute successfully.
```

Comment:
We have a rather long benchmarking notebook and a dice-solver notebook I'm not confident about running in CI, so I wanted to start by testing the two tutorials. As you alluded to, maybe I can just ignore the couple of problematic notebooks and run the rest.

Slightly related, I am going to write a round of unit tests, which will surely lead to a bug-fix release fairly soon. 0.4 should be a much more strictly-tested release